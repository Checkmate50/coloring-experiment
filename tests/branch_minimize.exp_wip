inner : int;
outer : int;
inner_res : Result;
res : Result;

inner < inner_res;
outer < res;

int;
if {
    int;
    Result;
}
else {
}
Result;