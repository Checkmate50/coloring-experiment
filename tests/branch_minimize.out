outer;
if {
    inner;
    inner_res;
}
else {
}
res;
